tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "compilerOptions": {
     "baseUrl": ".",
+    "strictNullChecks": false,
```

Comment:
```suggestion
```

**suggestion**: remove this line. We don't have `strict` mode set to `true` this be default will be `false` https://www.typescriptlang.org/tsconfig#strictNullChecks

Reply:
I've added an explanatory comment.